cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,117 @@
 /// <reference types='cypress' />
+const user = {
+  firstName: 'inna',
+  lastName: 'levytska',
+  email: 'inna123@gmail.com',
+  age: '40',
+  salary: '100',
+  department: 'QA'
+}
+beforeEach(() => {
+  cy.visit('https://demoqa.com/webtables')
+  });
 
 describe('Web Tables page', () => {
   it('', () => {
 
+
+  });
+  it('should have pagination', () => {
```

Comment:
you do not check the pagination here. You need to add workers to make workers amount equal to 6. Than Change rows count to 5. Click the button next -> assert that page 2 is displayed. Click button previous -> assert that page 1 is displayed. Also in this test you can check the rows count. When you change row count to 5 -> assert that table contain 5 rows.

Reply:
I didn't see this condition in our homework "You need to add workers to make workers amount equal to 6. Than Change rows count to 5. Click the button next -> assert that page 2 is displayed. Click button previous -> assert that page 1 is displayed. Also in this test you can check the rows count. When you change row count to 5 -> assert that table contain 5 rows."
Our tasks bellow:
1. Pagination.
1. Rows count selection.
1. Add a new worker.
1. Delete a worker.
1. Delete all workers.
1. Find a worker in the search field and edit it.
1. Validate data in the worker row after editing the worker.
1. Check the search by all column values.
And everything is done.

Reply:
how you can test pagination if you do not check the pagination buttons functionality?)

Reply:
I check pagination with the next steps: .should('contain', 'Previous')
      .and('contain', 'Next');
    cy.contains('.-pageInfo', 'Page')
      .should('exist');
I can't see condition that I have to check functionality.

Reply:
here is condition
<img width="477" alt="image" src="https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/85580286/7e12f5ab-d337-442b-a702-488e5a48c33b">

Reply:
Still I belive that my first test follow condition